src/content/api/compilation-hooks.md

```diff
@@ -197,7 +197,7 @@ W> This hook will be removed in v5.0.0
 
 `SyncBailHook`
 
-Called at the beginning of the chunks optimizion phase. A plugin can tap into this hook to perform chunks optimizations.
+Called at the beginning of the chunks optimization phase. A plugin can tap into this hook to perform chunk optimizations.
```

Comment:
Same as above, multiple chunks coming in (0 to infinity)

Reply:
Optimization is specific to an individual chunk (so, current version in docs is also incorrect), thus I left a suggestion I feel that should be here.
```suggestion
Called at the beginning of the chunk optimization phase. A plugin can tap into this hook to perform optimizations on chunks.
```